tests/ColorSchemeTests.cs

```diff
@@ -121,26 +119,30 @@ public void TestColorSchemeProperty_ToString(bool testMultiSelectingSeveralTimes
             selection.SetSelection(p.Design);
             selection.Clear();
 
-            ClassicAssert.AreEqual(pink, p.Design.View.ColorScheme);
+            Assert.That( p.Design.View.ColorScheme, Is.EqualTo( pink ) );
         }
 
         selection.SetSelection(p.Design);
-        ClassicAssert.AreNotEqual(pink, p.Design.View.ColorScheme, "Expected view to be selected so be green not pink");
-        ClassicAssert.AreEqual("ColorScheme:pink", p.ToString(), "Expected us to know it was pink under the hood even while selected");
+        Assert.Multiple( ( ) =>
+        {
+            Assert.That( p.Design.View.ColorScheme, Is.Not.EqualTo( pink ), "Expected view to be selected to be green, not pink");
+            Assert.That( p.ToString(), Is.EqualTo( "ColorScheme:pink" ), "Expected us to know it was pink under the hood even while selected");
+        } );
```

Comment:
Love these changes!

Reply:
I like the fluet-like way NUnit went, too. Makes them read like a sentence, for nice and easy review and maintenance. Kinda hits you over the head with exactly what it's doing. 😅